src/reflect.js

```diff
@@ -63,3 +63,8 @@ if (typeof Reflect.construct !== 'function') {
     return new (bind.apply(Target, a));
   };
 }
+
+if (typeof Reflect.ownKeys !== 'function')
+    Reflect.ownKeys = function (o) {
+    return (Object.getOwnPropertyNames(o).concat(Object.getOwnPropertySymbols(o))));
```

Comment:
Okay, that threw me for a minute. IE11 does *not* implement Object.getOwnPropertySymbols .. which did confuse me for a minute as this is working in production for me as we speak. It turns out you have already implemented a polyfill for getOwnPropertySymbols [here]( https://github.com/aurelia/polyfills/blob/37331dc6470a133fb5e38d5c63eac671980af8a8/src/symbol.js#L1)